test/xpu_api/utilities/meta/meta.unary/meta.unary.prop/is_pod.pass.cpp

```diff
@@ -0,0 +1,95 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+
+#include "support/test_macros.h"
+#include "support/utils.h"
+#include "support/utils_invoke.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+template <class T>
+void
+test_is_pod(sycl::queue& deviceQueue)
+{
+    deviceQueue.submit([&](sycl::handler& cgh) {
+        cgh.single_task<T>([=]() {
+            static_assert(dpl::is_pod<T>::value);
```

Comment:
As far as I remember, in previous tests-related PRs all code related to `dpl::is_pod` was wrapped on macros to use it only until C++20 since it is now deprecated. 
Should we suppress some deprecation warnings in this test?

Reply:
Fixed. There are checked `TEST_STD_VER` define state.